content/en/blog/_posts/Hardware-Accelerated-SSLTLS-Termination-in-Ingress-Controllers-using-Kubernetes-Device-Plugins-and-RuntimeClass.md

```diff
@@ -0,0 +1,168 @@
+**Authors:** Mikko Ylinen (Intel)
+
```

Comment:
Hello @mythi.
- Does your file have front matter? Such as:
```yaml
---
layout: blog
title: 'Pod Priority and Preemption in Kubernetes'
date: 2019-04-16
---
```
- I was unable to see the preview of this file.
- I think `content\en\blog\_posts\future-of-cloud-providers.md` contains an image. May help for your changes here.

Reply:
@kbhawkey thanks for the review. I've added the front matter. Let's see if the preview works after that.

Reply:
I don't yet have the date for publication agreed. I've picked 04/23 and made changes (including the images) based on that. The preview should work now.